PyDMXControl/profiles/defaults/_Fixture.py

```diff
@@ -389,4 +402,7 @@ def clear_effects(self) -> 'Fixture':
         # Clear
         self.__effects = []
 
+        # Remove local callbacks - FixtureHelper
+        self.remove_local_callbacks()
```

Comment:
I don't think this should be called here? Effects are tracked on their own in `__effects`, and that's what this method deals with. I think clearly callbacks that're outside effects should be its own call that a user needs to do?

Reply:
Yeah I think that's a good idea. I'll make it a seperate call.